src/server/api/routes/status-box.router.js

```diff
@@ -22,4 +22,20 @@ router.get(
   },
 );
 
+// ENDPOINT: /api/status-box/daysTilHarvest/:batchId :GET to get days till harvest date of a batch with batchId
+router.get('/days-to-harvest/:batchId', (req, res, next) => {
+  statusBoxController
```

Comment:
the middleware checkIfAuthenticated is missing in this endpoint, may be you can refer to the : firebase-middlewares.md document in our project to on how to use it on end point :)